src/main/kotlin/com/pl/sggw/tinder/infrastructure/addUser/api/AddUserController.kt

```diff
@@ -0,0 +1,31 @@
+package com.pl.sggw.tinder.infrastructure.addUser.api
+
+import com.pl.sggw.tinder.domain.addUser.AddUserRepository
+import com.pl.sggw.tinder.domain.addUser.AddUserDto
+import io.swagger.annotations.Api
+import io.swagger.annotations.ApiOperation
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.OffsetDateTime
+
+
+@Api(value = "add-user", description = "Rest api for adding new user")
+@RequestMapping("/api/add-user")
+@RestController
+class AddUserController(val addUserRepository: AddUserRepository) {
+
+    @PostMapping("/add-user")
+    @ApiOperation("Add new tinder user")
+    fun addUser(@RequestBody request : AddUserRequest){
+        addUserRepository.addUser(
```

Comment:
Przy  dodaniu uzytkownika dobrze by bylo sprawdzac czy juz taki nie istnieje i jesli nei to odrzucac takie zadanie